docs/rapidstart.md

```diff
@@ -0,0 +1,717 @@
+
+## Configure Google as the Identity Provider
+
+Go to the [Google Cloud Console Resource Manager](https://console.cloud.google.com/cloud-resource-manager) to create a new project. 
+
+* Click on **Create Project** and name it (e.g, UC demo)
+
+* Go to your project, then select **APIs & Services \> OAuth consent screen**  
+  <img src="./assets/images/googleconsole-oauth-consent-screen.png" width=500>
+    
+* Choose **External** and then click **Create**.
+
+* For the **App information** dialog, please provide the *App name*, *user-supported* *email*, and *Developer contact information* at a minimum.  Click **Save and Continue**.
+
+* For the purpose of this demo, skip *Scopes,* *Test users*, and go directly to *Summary* return **Back to Dashboard**.
+
+
+* Click **APIs & Services \> Credentials** and click the **Create Credentials** button to create an **OAuth client ID**.  
+  * **Application type**:  Web application  
+  * **Name**: UC demo web client  
+  * **Authorized JavaScript origins**: http://localhost, http://localhost:3000 (this is required for the UI)  
+  * **Authorized redirect URIs**: http://localhost, http://localhost:3000 (this is required for the UI)
+
+  ![](./assets/images/googleconsole-create-oauth-client-id.png)
+
+* Once the OAuth client ID is created, a pop-up dialog box will appear containing the **Client ID** and **client secret**. Make sure to copy and/or download it (via the Download JSON button), as this will be needed in the following steps.
+
+  <img src="./assets/images/googleconsole-oauth-client-created.png" width=400 />
+
+## Configure UC Server Settings
+
+Start by editing `etc/conf/server.properties` with the aforementioned **Client ID** and **Client secret**.
+
+```bash
+server.authorization=enable
+server.authorization-url=https://accounts.google.com/o/oauth2/auth
+server.token-url=https://oauth2.googleapis.com/token
+server.client-id=<Client ID provided earlier>
+server.client-secret=<Client secret provided earlier>
+```
+
+## Restart the UC Server
+
+Now that the Google Authentication is configured, restart the UC Server with the following command. 
+
+```bash
+bin/start-uc-server
+```
+
+Behind the scenes, on startup, the UC server will configure itself with the following:
+
+* Creation of internal signing keys and authentication files in `etc/conf`  
+* Creation of an admin access token in etc/conf/token.txt  
+* Creation of an admin account in the user database  
+* Granting the admin account as the metastore admin for the server
+
+
+## Testing User and Admin Authentication
+
+The following steps allow you to test user and admin authentication to your local Unity Catalog.
+
+### Use admin token to verify admin user is in local database
+
+As noted earlier, on startup the UC server configured itself and created the admin access token at `/etc/conf/token.txt`.  The following command uses the same admin authentication token to view the user list.
```

Comment:
nit: remove leading / in /etc/conf/token.txt

Reply:
Ack!